src/vmm/src/vmm_config/boot_source.rs

```diff
@@ -32,7 +30,6 @@ pub struct BootSourceConfig {
     pub initrd_path: Option<String>,
     /// The boot arguments to pass to the kernel. If this field is uninitialized,
     /// DEFAULT_KERNEL_CMDLINE is used.
-    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
mh, why remove this? Or rather, why was this here in the first place?

Actually, why did these already implement Serialize anyway. They're for the API, so only Deserialize should've been needed :thinking: